### ISSUE TITLE
Rework profile view

### ISSUE THREAD

**Issue by Echolon** [kriztan/Pix-Art-Messenger#390]
Hi Chris,

I have some additional suggestions about the profile view, one I already told you about. Here we go:

This is what you currently see when you open you own profile:

<img src="https://user-images.githubusercontent.com/29168011/65080619-05fe6c00-d9a2-11e9-8ace-2207b715469f.png" width="270">
I like the idea that people can see their full-screen avatar. Unfortunately, they get the impression that there is nothing else below. But indeed there is more information. Therefore, I suggest to show the avatar as a small icon and offer the possibility to scale up (by click and show a small arrow in the lower-right corner).

While the small icon is shown there can also be more information placed and further implicitly told that there is more information available (by scrolling).
This also brings me to the next two suggestions: There should be shown what the profile's status is (also empty) and the status about battery optimization. I already suggested to Chris that this should be actively checked by the app, as well as reminded to the user once a month, and additionally once a week in the first two week after the initial installation.

To summarize:

**1. Enable scaled avatar and scale down by default**
**2. Show the profile status**
**3. Show a reminder about the battery optimization to remind people**
**4. Maybe make also some recommendations if presence etc is not set**

Thanks for your efforts Chris and looking forward!
